tests/handler.test.js

```diff
@@ -1,12 +1,59 @@
-import * as handler from '../handler';
+import * as redis from '../src/utils/redis';
+import * as fixtures from "./fixtures";
+import { sendToS3 } from '../src/utils/s3';
+import pdf from '../src/chrome/pdf';
+import { htmlToPDFAsync } from '../handler';
 
-test('hello', async () => {
-  const event = 'event';
-  const context = 'context';
-  const callback = (error, response) => {
-    expect(response.statusCode).toEqual(200);
-    expect(typeof response.body).toBe("string");
-  };
+jest.mock('../src/utils/redis');
+jest.mock('../src/chrome/pdf');
+jest.mock('../src/utils/s3');
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+test('htmlToPDFAsync() handler should execute with success', async () => {
+  const buf = Buffer.from([0x62, 0x75, 0x66, 0x66, 0x65, 0x72]);
+  redis.getWithJsonParse.mockReturnValue((Promise.resolve(fixtures.pdfBody())));
```

Comment:
Dentro desse async, não daria pra resolver as promises com await?
```suggestion
  redis.getWithJsonParse.mockReturnValue((await fixtures.pdfBody()));
```

Reply:
Vou testar aqui, valeu !